src/server/wsServer.ts

```diff
@@ -178,15 +182,21 @@ export class WSServer extends Disposable {
 				type: 'ws',
 				err: err,
 			});
-			console.log(`Unknown error: ${err}`);
+			console.log(localize('unknownError', 'Unknown error: {0}', err));
 		}
 	}
 
 	/**
 	 * @description handle the websocket successfully connecting.
 	 */
 	private handleWSListen(): void {
-		console.log(`Websocket server is running on port ${this._wsPort}`);
+		console.log(
+			localize(
+				'websocketRunningOnPort',
+				'Websocket server is running on port {0}',

```

Comment:
if this is something that only you, as the maintainer, needs to see, you can keep it in English. However, if you want your users to see this, then leave it localized.

Reply:
I was originally thinking that it would be a good idea since there's the browser option to open devtools, but thinking back, it seems sorta niche. will change back